rest-service/manager_rest/resource_manager.py

```diff
@@ -483,6 +483,37 @@ def delete_deployment(self,
         else:
             return self.sm.delete(deployment)
 
+    def resume_execution(self, execution_id):
+        execution = self.sm.list(
+            models.Execution, filters={'id': execution_id},
+            get_all_results=True)[0]
+        if execution.status in ExecutionState.END_STATES or \
```

Comment:
Is there a reason this wouldn't be better checking for only allowed states and then raising an error stating what states one is allowed to resume an execution in?

Reply:
with this PR, let's not, but stick with the "if it works, it works". With another PR, we'll be adding resume just for cancelled and failed